test/packages/false_positives/httpjson_false_positive_asserts/_dev/build/build.yml

```
@@ -0,0 +1,3 @@
+dependencies:
+  ecs:
+    reference: git@8.8
```

Let's just use tag here. 
There were some issues related to use branches instead of tags

```suggestion
    reference: git@v8.8.0
```

Probably, it results in some failures in the README